var/spack/repos/builtin/packages/jellyfish/package.py

```diff
@@ -18,7 +18,27 @@ class Jellyfish(AutotoolsPackage):
     version('1.1.11', sha256='496645d96b08ba35db1f856d857a159798c73cbc1eccb852ef1b253d1678c8e2',
             url='https://www.cbcb.umd.edu/software/jellyfish/jellyfish-1.1.11.tar.gz')
 
+    variant('pybind', default=False, description='Enable python binding')
+    variant('rubybind', default=False, description='Enable ruby binding')
+    variant('perlbind', default=False, description='Enable perl binding')
+
     depends_on('perl', type=('build', 'run'))
     depends_on('python', type=('build', 'run'))
 
     patch('dna_codes.patch', when='@1.1.11')
+
+    def configure_args(self):
+        spec = self.spec
+        if '+pybind' in spec:
+            args = [
+                '--enable-python-binding',
+            ]
+        elif '+rubybind' in spec:
+            args = [
+                '--enable-ruby-binding',
+            ]
+        elif '+perlbind' in spec:
+            args = [
+                '--enable-perl-binding'
+            ]
```

Comment:
Are the bindings exclusive, i.e. can't we activate them for each language?

Reply:
Ah, yes, I believe you are probably correct, I will set some time aside to have a look.